src/core/analytics.js

```diff
@@ -404,57 +408,99 @@ class Analytics {
     }
   }
 
+  /**
+   * A function that gets the transformed event and
+   * depending on the response either send the transformed/untransformed event to destination or just log error.
+   * @param {Array} destWithTransformation   List of device mode destination that has transformation connected
+   * @param {Object} rudderElement           Rudder event
+   * @param {string} methodName              Type of event page/track/identify etc.
+   */
   sendTransformedDataToDestination(destWithTransformation, rudderElement, methodName) {
     try {
       // convert integrations object to server identified names, kind of hack now!
       transformToServerNames(rudderElement.message.integrations);
-
+      const destinationIds = destWithTransformation.map((e) => e.destinationId);
```

Comment:
does destWithTransformation also take into acocunt event filtering for the destination?

Reply:
Yes